taprpc/universerpc/universe.proto

```diff
@@ -169,6 +169,13 @@ service Universe {
     */
     rpc UpdateSupplyCommit (UpdateSupplyCommitRequest)
         returns (UpdateSupplyCommitResponse);
+
+    /* tapcli: `universe fetchsupplycommit`
```

Comment:
We need actual command counterparts for all three RPCs in the `cmd/tapcli` package.